decrypt/decrypt_test.go

```diff
@@ -0,0 +1,165 @@
+package decrypt
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+	"testing"
+
+	createKey "github.com/NBISweden/sda-cli/create_key"
+	"github.com/NBISweden/sda-cli/encrypt"
+	"github.com/NBISweden/sda-cli/helpers"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+)
+
+type DecryptTests struct {
+	suite.Suite
+	tempDir     string
+	fileContent []byte
+	testFile    *os.File
+}
+
+func TestDecryptTestSuite(t *testing.T) {
+	suite.Run(t, new(DecryptTests))
+}
+
+func (suite *DecryptTests) SetupTest() {
+
+	var err error
+
+	// Create a temporary directory for our files
+	suite.tempDir, err = ioutil.TempDir(os.TempDir(), "sda-cli-test-")
+	if err != nil {
+		log.Fatal("Couldn't create temporary test directory", err)
+	}
+
+	// create a test file...
+	suite.testFile, err = ioutil.TempFile(suite.tempDir, "testfile-")
+	if err != nil {
+		log.Fatal("cannot create temporary public key file", err)
+	}
+
+	// ... create some content ...
+	suite.fileContent = []byte("This is some fine content right here.")
+
+	// ... and write the known content to it
+	err = ioutil.WriteFile(suite.testFile.Name(), suite.fileContent, 0600)
+	if err != nil {
+		log.Fatalf("failed to write to testfile: %s", err)
+	}
```

Comment:
Use asserts in tests instead of log.fatal

Reply:
I used Log.Fatal when there were failures unrelated to the tests, but I can replace them with asserts instead :)